src/olympia/amo/management/commands/data_seed.py

```diff
@@ -13,56 +10,25 @@ class Command(BaseDataCommand):
         'generated add-ons, and data from AMO production.'
     )
 
-    def _clean_storage(self, root: str, dir_dict: dict[str, str | dict]) -> None:
-        for key, value in dir_dict.items():
-            curr_path = os.path.join(root, key)
-            if isinstance(value, dict):
-                self._clean_storage(curr_path, value)
-            else:
-                shutil.rmtree(curr_path, ignore_errors=True)
-                os.makedirs(curr_path, exist_ok=True)
-
-    def clean_storage(self):
-        self.logger.info('Cleaning storage...')
-        self._clean_storage(
-            settings.STORAGE_ROOT,
-            {
-                'files': '',
-                'shared_storage': {
-                    'tmp': {
-                        'addons': '',
-                        'data': '',
-                        'file_viewer': '',
-                        'guarded-addons': '',
-                        'icon': '',
-                        'log': '',
-                        'persona_header': '',
-                        'preview': '',
-                        'test': '',
-                        'uploads': '',
-                    },
-                    'uploads': '',
-                },
-            },
-        )
-
     def handle(self, *args, **options):
         num_addons = 10
         num_themes = 5
 
+        # Delete any existing data_seed backup
         self.clean_dir(self.data_backup_init)
 
         self.logger.info('Resetting database...')
-        call_command('flush', '--noinput')
+        call_command('reset_db', '--no-utf8', '--noinput')
+        # Delete any local storage files
+        # This should happen after we reset the database to ensure any records
+        # relying on storage are deleted.
         self.clean_storage()
-        # reindex --wipe will force the ES mapping to be re-installed.
-        call_command('reindex', '--wipe', '--force', '--noinput')
+        # Migrate the database
         call_command('migrate', '--noinput')
 
         self.logger.info('Loading initial data...')
         call_command('loaddata', 'initial.json')
         call_command('import_prod_versions')
-        call_command('import_licenses')
```

Comment:
Can you link to the pr/issue where we made this command redundant?  (I thought Baku's work was still in progress)

Reply:
<https://github.com/mozilla/addons-server/pull/22818>

Reply:
@KevinMind this is the patch that added the line, 2 days ago.  How is it redundant after 2 days - it's not even be deployed to stage yet?